spec/features/super_admin/lookup_wifi_user_contact_details_spec.rb

```diff
@@ -0,0 +1,60 @@
+describe 'Lookup wifi user contact details', type: :feature do
+  let(:user) { create(:user, :super_admin) }
+  let(:search_term) { '' }
+  let(:contact) { 'wifi.user@govwifi.org' }
+
+  after { WifiUser.destroy_all }
```

Comment:
I think this will automatically be cleaned up by RSpec, or is it specifically needed to avoid errors?

Reply:
I found I was getting duplicate key errors on username. I agree it's a bit of a smell that something isn't right in the setup.

Reply:
👍